batch/python/tiles_geojson.py

```diff
@@ -0,0 +1,96 @@
+import json
+from concurrent.futures import ProcessPoolExecutor, as_completed
+from typing import Any, Dict, List, Tuple
+
+from geojson import Feature, FeatureCollection
+from pyproj import CRS, Transformer
+from shapely.geometry import Polygon
+from shapely.ops import unary_union
+
+from errors import GDALError
+from gdal_utils import run_gdal_subcommand
+
+
+def to_4326(crs: CRS, x: float, y: float) -> Tuple[float, float]:
+    transformer = Transformer.from_crs(
+        crs, CRS.from_epsg(4326), always_xy=True
+    )
+    return transformer.transform(x, y)
+
+
+def extract_metadata_from_gdalinfo(gdalinfo_json: Dict[str, Any]) -> Dict[str, Any]:
+    """Extract necessary metadata from the gdalinfo JSON output."""
+    corner_coordinates = gdalinfo_json["cornerCoordinates"]
+
+    crs: CRS = CRS.from_string(gdalinfo_json["coordinateSystem"]["wkt"])
+    metadata = {
+        # NOTE: pixetl seems to always write features in tiles.geojson in
```

Comment:
<img width="521" alt="Screenshot 2024-12-11 at 1 43 40 PM" src="https://github.com/user-attachments/assets/64937194-1ef8-4589-b6b3-5ba92c312778" />
<img width="768" alt="Screenshot 2024-12-11 at 1 43 10 PM" src="https://github.com/user-attachments/assets/83a6faf1-e4a7-4c64-8754-b9bce3bd4d2c" />
I admit I'm a little bit fuzzy on the terminology, but I think technically EPSG:3857 and EPSG:4326 are both WGS 84? In any case, what I really mean is that the coordinates in the feature are always in degrees, even for WM tiles. I *think* they should be in meters for WM tiles. So that's what I'm trying to duplicate (even though it feels wrong).

Reply:
I'll re-write the comment to clarify that it's the units, not the actual projection.